File: Module5/lab/5.5/ex01.js
const prompt = require('prompt-sync')();

var num = 0, perfectsquare = false;

num = Number(prompt("Enter number: "))

if (num>0 && num<=100) {
    for (i=0; i<11; i++) {
        num == i * i;
        perfectsquare = true;
    }
    if (perfectsquare = true) {
        console.log(num + ' is a perfect square');
    }
    else {
        console.log(num + ' is NOT a perfect square');
    }
}
else {
    console.log ('Please enter number between 1-100')
}

File: Module6/lab/6.4/ex01.py
num = int(input('Please enter a positive number: '))

if num > 0:
    for i in range(1,11):
        results = num * i
        print (f'{num} x {i} = {results} ')
        # print('{} x {} = {}'.format(num, i, num * i))
else:
    print('Invalid integer')